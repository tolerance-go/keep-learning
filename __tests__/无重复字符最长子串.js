const lengthOfLongestSubstring = require("../src/算法/无重复字符最长子串");

test(`lengthOfLongestSubstring("aab")`, () => {
  expect(lengthOfLongestSubstring("aab")).toMatchSnapshot();
});
