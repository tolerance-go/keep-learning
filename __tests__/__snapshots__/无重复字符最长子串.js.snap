// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lengthOfLongestSubstring("aab") 1`] = `2`;
